trashtalk/google_sheets.py

```diff
@@ -112,10 +114,10 @@ def send_to_sheet(id, tool_data):
                       standard_weed_wrench,large_weed_wrench,plastic_bag,green_waste,reusable_bag,first_aid,cooler,
                       five_gal_buck,two_gal_buck,skimmer, ed_poster, other]
 
-    print("Formatted Data:", formatted_data)
+    #print("Formatted Data:", formatted_data)
```

Comment:
In general, these print statements would be logged at the `DEBUG` level. That way can use it during testing and debugging and not have to worry about it otherwise. Just FYI